end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -581,25 +624,28 @@ We can proceed to apply this algorithm to the entire dataset, and then extract
 the EICs for our two test ions to confirm that the process has been successful.
 
 ```{r run find chrompeak on entire dataset}
+#' Using the same settings, but with default snthresh
+param <- CentWaveParam(peakwidth = c(1, 8), ppm = 15, integrate = 2)
+
 data <- findChromPeaks(data, param = param, chunkSize = 5)
 
-#' Test if we find cystine and 1- Methylhistidine again 
-eic_cystine <- chromatogram(data, mz = cystine_mz + c(-0.05, 0.05),
-                           rt = c(205, 215))
+#' Test if we find cystine and 1- Methylhistidine again
+eic_cystine <- chromatogram(data, mz = cystine_mz + c(-0.01, 0.01),
```

Comment:
for consistency (and also because the results don't changes) I would keep 0.01 since we use that at many different places.